microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -3,254 +3,284 @@ description: The latest experimental features in Microsoft Edge DevTools
 ---
-# Experimental features  
+# Experimental features
 
-Microsoft Edge DevTools provide access to experimental features that are still in development.  You may test and [provide feedback](#providing-feedback-on-experimental-features) before each feature is released.  
+Microsoft Edge DevTools provide access to experimental features that are still in development.  You can test and [provide feedback](#providing-feedback-on-experimental-features) before each feature is released.
 
-While experimental features are available in all channels of Microsoft Edge, you may get the latest experimental features using the Microsoft Edge Canary channel.  
+Experimental features are available in all of the preview channels of Microsoft Edge.  You can get the latest experimental features by using the Microsoft Edge Canary channel.  This article only covers selected experimental features.  For the full list, see the **Settings** > **Experiments** page in DevTools.
 
-## Turn on experimental features  
 
-To turn on \(or off\) experimental features in Microsoft Edge, complete the following steps.  
+<!-- ====================================================================== -->
+## Experimental features which are turned on by default
 
-1.  [Open DevTools][DevtoolsOpenIndex].  
-    *   Select `Control`+`Shift`+`I` \(Windows, Linux\) or `Command`+`Option`+`I` \(macOS\).  For more information, navigate to [Microsoft Edge DevTools keyboard shortcuts][DevtoolsShortcutsIndex].  
-1.  Open the [Settings][DevToolsCustomizeIndexSettings] pane.  
-    *   Select `Shift`+`?`.  For more information, navigate to [Microsoft Edge DevTools keyboard shortcuts][DevtoolsShortcutsIndex].  
-1.  On the left side of the **Settings** pane, choose the **Experiments** section.  
-    
-    :::image type="complex" source="../media/experiments-devtools.msft.png" alt-text="The Experiments page in Settings" lightbox="../media/experiments-devtools.msft.png":::
-       The **Experiments** page in **Settings**  
-    :::image-end:::  
-    
-1.  On the **Experiments** page, scroll through the list of all available experimental features and choose the checkbox next to each feature that you want to test.  
-1.  Close and reopen Microsoft Edge DevTools.  
-    
-> [!NOTE]
-> Experimental features are constantly being updated and may cause performance issues.  To turn off an experimental feature, open the **Experiments** page and clear the checkbox of the experimental feature that you want to turn off.  
+As of Microsoft Edge 94, the following experimental features are turned on by default:
```

Comment:
What do you think about adding this at the end of the sentence: ", this means that they can be used right away, but can be turned off if needed".